src/components/POLine/Cost/CostForm.js

```diff
@@ -30,7 +30,7 @@ export const requiredPositiveQuantity = (value) => {
     : <FormattedMessage id="ui-orders.cost.validation.shouldBePositive" />;
 };
 
-const validateRequiredNotNegative = (value) => {
+export const validateRequiredNotNegative = (value) => {
```

Comment:
as for me it's not correct util from form to form, please move it to separate file

Reply:
validators were moved in separate file